jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java

```diff
@@ -95,6 +95,13 @@ private List<TarArchiveEntry> getSortedEntries() {
     }
   }
 
+  private static void clearTimeHeaders(TarArchiveEntry entry) {
```

Comment:
I think you should also call this after line 83?

Reply:
Maybe name it `clearPaxTimeHeaders()`?

Reply:
Yes that's probably wise!